Plugins/IntelliJ/gradle.properties

```diff
@@ -3,9 +3,9 @@
 
 pluginGroup = com.shopify.testify
 pluginName = Android Testify - Screenshot Instrumentation Tests
-pluginVersion = 0.1.2
 pluginSinceBuild = 193
 pluginUntilBuild = 211.*
+javaVersion = 1.8
 
 platformType = IC
 platformVersion = 2020.1
```

Comment:
Looks like this version has problems building on `aarch64`. I'll log an issue for looking adding M1 support.